activiti-cloud-service-common/activiti-cloud-services-common-identity-keycloak/src/test/java/org/activiti/cloud/services/identity/keycloak/KeycloakClientIT.java

```diff
@@ -313,4 +314,12 @@ public void should_getGroupsClientRoleMapping() {
         assertThat(groups).extracting("name").contains("salesgroup");
     }
 
+
+    @Test
+    public void should_getUsers_by_groupId() {
+        List<KeycloakUser> users = keycloakClient.getUsersByGroupId(HR_GROUP_ID);
+
+        assertThat(users).hasSizeGreaterThanOrEqualTo(1);
```

Comment:
could we check at least the name of the user? (I assume it is `hruser` right?)

Reply:
also, `assertThat(users).hasSize(1)` should be more accurate to what we want to check

Reply:
There's a couple of issues about that.

1. Based on configuration `activiti-realm.json` I know there are at least one user in HR_GROUP. In addition, on group HR_GROUP are performed some logic in other tests. So the only thing I can test is hasSizeGreaterThanOrEqualTo
![image](https://user-images.githubusercontent.com/5747095/187647446-8928298c-22a0-412e-85d7-645c80b34dc4.png)

2. Unfortunately, as showed in previous image, the other groups haven't got id

If necessary I can create a new group and new user only for testing this issue.

Reply:
I see, now that I remember hruser and hradmin are inside that group, and maybe more
okay, it is safe to check it in the way you did
thanks for the clarification